modules/rtm/R/resample.R

```diff
@@ -0,0 +1,57 @@
+#' Resample vector, matrix, or spectra
+#'
+#' Conveinent wrapper around base R's `splinefun`. See [stats::splinefun()] 
```

Comment:
Should we enable linear as well?

Reply:
Yeah, I suppose that could be useful. I'll add a `if (method %in% c("linear", "constant")) <use approxfun> else <use splinefun>`.